back/spec/services/invites_service_spec.rb

```diff
@@ -355,4 +355,25 @@
       end
     end
   end
+
+  describe 'bulk_create' do
+    let(:test_email1) { 'find_me_1@test.com' }
+    let(:test_email2) { 'find_me_2@test.com' }
+    let(:hash_array) do
+      [
+        { 'email' => test_email1 },
+        { 'email' => test_email2 }
+      ]
+    end
+    let(:inviter) { create(:user) }
+
+    context 'with multiple emails and no names' do
+      it 'creates users with no slugs' do
+        service.bulk_create(hash_array, _default_params = {}, inviter)
```

Comment:
Why do you declare `_default_params` variable here? You can just pass it like `bulk_create(hash_array, {}, inviter)`

Reply:
Just to help the reader understand, since it is the var name in the method and is nicely descriptive.

Reply:
Nice trick!